x-pack/plugins/maps/public/connected_components/layer_panel/join_editor/resources/join.js

```diff
@@ -192,8 +192,7 @@ export class Join extends Component {
           {metricsExpression}
 
           {whereExpression}
-
-          {globalFilterCheckbox}
+          <EuiFlexItem>{globalFilterCheckbox}</EuiFlexItem>
```

Comment:
Would you mind moving `EuiFlexItem` to where `globalFilterCheckbox` is defined on line 151?